### DISCUSSION TITLE
Content Queries

### DISCUSSION THREAD

**Discussion by jhorii** [statamic/cms#4554] [Q&A] [answered]
I'm trying to lookup some information on queries and the new documentation has me completely confused. [https://statamic.dev/content-queries](https://statamic.dev/content-queries)

What I don't know from this page:

1. What type of files do I run these queries on? And where do I put them?  Is this in a view model?  On an antlers template?
2. How do I access the results of the query in my antlers templates?

Any help would be appreciated.

**Comment by jasonvarga** [comment, marked as answer]
Where you put this code depends on what you're trying to do.

If you're trying to do stuff in your template, the most likely place would be within a tag. https://statamic.dev/extending/tags

**Reply by jhorii** [reply]
Thanks Jason.

I have a follow up question but I'll post as a separate question.
